org.eclipse.scanning.test/src/org/eclipse/scanning/test/event/queues/mocks/MockPositioner.java

```diff
@@ -62,17 +62,17 @@ public IPosition getPosition() throws ScanningException {
 	}
 
 	@Override
-	public List<IScannable<?>> getMonitors() throws ScanningException {
+	public List<IScannable<?>> getMonitorsPerPoint() throws ScanningException {
 		return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)